packages/shared/routes/dashboard/governance/Governance.svelte

```diff
@@ -1,21 +1,46 @@
 <script lang="typescript">
-    import { DashboardPane } from 'shared/components'
-    import { GovernanceHeader, GovernanceVotingPower, GovernanceInfo, GovernanceEvents } from './views'
+    import { WalletPill } from 'shared/components'
+    import { GovernanceHeader, GovernanceDashboard, GovernanceEventDetails } from './views'
+    import { GovernanceRoutes } from 'shared/lib/typings/routes'
+    import { governanceRoute } from 'shared/lib/router'
+    import { participationEvents } from 'shared/lib/participation/stores'
+    import { wallet } from 'shared/lib/wallet'
+    import { get } from 'svelte/store'
+    import { selectedAccountId } from 'shared/lib/wallet'
+    import type { Locale } from 'shared/lib/typings/i18n'
+
+    export let locale: Locale
```

Comment:
I would recommend using the function `localize` instead of using the prop `locale`, we have started updating that here and there. I would suggest that new developments also adopt it 🙏🏼  eg https://github.com/iotaledger/firefly/blob/develop/packages/shared/components/SettingsNavigator.svelte#L3